helm-chart/values.yaml

```diff
@@ -91,6 +91,9 @@ cvat:
             name: http
     defaultStorage:
         enabled: true
+        storageClassName: default
+        accessModes:
+         - ReadWriteOnce
```

Comment:
Correct me if I am wrong, I was testing the cvat helm chart out today, and I believe this should probably be ReadWriteMany as at-least the original chart before your changes would spin up multiple pods seeking to access the same volume by PVC name. I dont know how cvat acts when multiple pods can read and write at once, but without this set to ReadWriteMany only one of the pods like utils or server will get this volume and the rest will be waiting for it indefinitely since they cant bind to it any more.

Reply:
Hi, I have limited knowledge of what is happening internally, so I left it as it is and I'm waiting for input from @azhavoro or @mdacoca to see if it's okay to change it by default.

In my opinion, the current chart may not work as it is, and we might need to change the default access mode. However, I'm almost certain that it would not help because the default storage class usually does not support this access mode either.

P.S. Thank you very much for reviewing! I appreciate it.

Reply:
Ah I see, for me I back my kubernetes cluster with rook so I have a default storage class that supports RWX but yeah I get what you mean, there does not seem to be a good solution for everyone but I think RWO is broken for everyone.

Also on a related note I might not assume the users default storage class, In my case I use [rook](https://rook.io/) which gives me these:
```
NAME                        PROVISIONER                     RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
ceph-block                  rook-ceph.rbd.csi.ceph.com      Delete          Immediate           true                   26h
ceph-bucket                 rook-ceph.ceph.rook.io/bucket   Delete          Immediate           false                  26h
ceph-filesystem (default)   rook-ceph.cephfs.csi.ceph.com   Delete          Immediate           true                   26h
```
Im not 100% certain but I believe if in the PVC you dont specify the storage class that it will auto populate it with whatever the cluster has as a default no matter what it is called. This might be more universal if you put this behind and if condition so if the user sets a storageClassName it will use it, and if not it will leave it blank in the CRD for kubernetes to fill it in. Something like {{ if eq .Values.cvat.backend.defaultStorage.storageClassName "" }} maybe or {{ if empty .Values.cvat.backend.defaultStorage.storageClassName }} (this was only quick off the top of my head may need editing the condition to invert it).

Also I wanted to say its no problem, I have a vested interest in this too so I would love to see your PR be merged as it works a lot better than the current one which is a bit broken in a few places.

Reply:
Thanks a lot, added)

Reply:
That's beautiful thanks for the changes!!! This now has made ArgoCD happier as well since it understands the value is now auto-filled by kube. Thanks a lot for your additions, super appreciated!
![cvat](https://github.com/opencv/cvat/assets/10534713/04ae7c47-6162-4e0c-b8c0-ecff8ac9d061)
I will keep testing the chart for you and let you know if there are any other minor changes but it all looks good currently.